### ISSUE TITLE
`// swiftformat:disable` doesn't affect extensionAccessControl

### ISSUE THREAD

**Issue by cleverer** [nicklockwood/SwiftFormat#771]
The following example:
```swift
// swiftformat:disable all
public protocol Foo {
	func test()
}

internal protocol Bar: Foo {}

extension Bar {
	public func test() {
		// …
	}
}
```
gets formatted to:
```swift
// swiftformat:disable all
public protocol Foo {
	func test()
}

internal protocol Bar: Foo {}

public extension Bar {
	func test() {
		// …
	}
}

```

which unfortunately breaks compilation (as `test()` needs to be public to satisfy `Foo`).

**Comment by nicklockwood** [comment]
Thanks for reporting, I'll get it fixed

**Comment by Speakus** [comment]
also `--disable extensionacl` inside `.swiftformat` does not affect the swiftformat logic

**Comment by nicklockwood** [comment]
@Speakus try `--disable extensionAccessControl` instead.

**Comment by nicklockwood** [comment]
@cleverer fixed in 0.47.2
